### ISSUE TITLE
wsl dpkg error

### ISSUE THREAD

**Issue by DIVINE1999** [Microsoft/WSL#3201]
divine@divine-pc:/mnt/c/Users/User$ sudo dpkg --configure -a
dpkg: error: parsing file '/var/lib/dpkg/updates/0037' near line 0:
 newline in field name '#padding'
i am not able to download any packages.At last it shows the above error message
pls help me
i've downloaded gedit but not able to open it.
gedit&

the same error occurs again

**Comment by froggx** [comment]
I think something got changed in a recent update because I ran into a similar problem yesterday trying to set up Kali in WSL on a fresh Win 10 Pro install (insider fast track, Build17134.48, somehow lagging behind on insider build updates).  This configuration was all something I've done at least 30 times prior to this update with no problems.  What's the term for that kind of thing, regression?

First off, in the CLI I was able to download and install my usual clusterf@#$ of files with the eternally awesome "apt-get install" command ("apt" is my goto for CLI package management, gotta love those super cow powers).

BTW, I'm one of those people that, ever since Windows 3.1 dropped, avoids touching a CLI like it's a plague rat of doom, so I should mention I run "vcxsrv" in multiwindow mode to keep an Xserver going so I can plunge into that gooey GUI goodness.

Anyway, using super cow pow... erm..."apt-get install," I had installed "synaptic" and went about my normal WSL Kali setup procedures.  Everything was going fine, the packages I needed downloaded from the repos with no trouble, and then "synaptic" starts up "dpkg --configure -a" to unpack and install and it would just sit there and eat up CPU cycles (fully loading all my cores).

Back in the CLI window (they lurk in the background and complain when they launch a graphical window), I was getting an error message regarding a failure to  run "dbus-launch" each time synaptic fired up "dpkg" (sorry, I don't remember the exact message and I'm not really feeling the urge to wreck things and recreate it atm).

TL;DR 
I was able to fix this by using "apt-get install" to download and install "debhelper" and "dbus-x11."
After that "dpkg --configure -a" worked fine for me both in "synaptic" and in the CLI. 

I don't know why "debhelper" fixed anything or why I had a "dbus" related problem to start with, but by that point I had been bashing my head into a wall for a good 30 minutes (which is long enough that I'm going to have to patch the sheetrock and repaint the wall), so I didn't really care anymore since my stuff worked.

I hope the fix I stumbled on helps you transcend your similar sounding issue. 
(Sorry if the readability is lacking, in my undergrad I took WAY too many classes where I had to stretch the length of research papers and ended up stuck with the "--verbose" argument permanently enabled in my head.)
